src/main/java/frc/robot/RobotContainer.java

```diff
@@ -72,11 +75,7 @@ public RobotContainer() {
     private void configureButtonBindings() {
         driverStick = new Joystick(0);
         controlStick = new Joystick(1);
-        
-        
-        // Instantiate the intake command and bind it 
-        intakeCommand = new OperatorIntakeCommand();
-        
+     
```

Comment:
Any reason this code has been removed?

Reply:
Justin asked me to remove this code as it was not being used.